servicetalk-examples/http/http2/src/main/java/io/servicetalk/examples/http/http2/alpn/HttpUrlClientWithAlpn.java

```diff
@@ -39,7 +39,7 @@ public static void main(String[] args) throws Exception {
         try (BlockingHttpClient client = HttpClients.forMultiAddressUrl().initializer((scheme, address, builder) -> {
             // If necessary, users can also take `address` into account for setting distinct protocols or TLS
             // configurations for various server addresses.
-            if ("https".equalsIgnoreCase(scheme)) {
+            if ("https".equals(scheme)) {
```

Comment:
How do you feel about changing examples and clarifying the contract in `MultiAddressHttpClientBuilder` javadoc?
The only risk is that examples can be copy pasted and if users messed up and used a non-lowercase constant on the left hand side, they will have a problem. It might be safer to teach them to use `equalsIgnoreCase`.

Reply:
Good observation re copy pasting problems. There is a reasonable chance people are using constants and they don't actually know what the string is. If I had to pick, I'd say leave it as `.equalsIgnoreCase` since as you noted, it's safer and still correct, if perhaps redundant.

Reply:
Ok, I will revert examples then